packages/headless/src/controllers/core/facets/facet-search/facet-search.ts

```diff
@@ -22,6 +19,15 @@ export interface GenericFacetSearchProps<T extends FacetSearchState> {
   options: FacetSearchOptions;
   getFacetSearch: () => T;
   isForFieldSuggestions: boolean;
+
+  executeFacetSearchActionCreator: (
+    facetId: string
+  ) => AsyncThunkAction<unknown, string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Why so many anys?

Reply:
Because the expected type for the third type variable of the `AsyncThunkAction` generic type extends the `AsyncThunkConfig` Redeux interface, which only has optional properties, so basically anything goes anways.

Reply:
I can use AsyncThunkOptions<unknown, ThunkExtraArguments> instead though, I think this will work!